tests/unit/single_table/efficacy/detection/test_detection.py

```diff
@@ -0,0 +1,64 @@
+from unittest.mock import patch
+
+import pandas as pd
+
+from sdmetrics.single_table import LogisticDetection
+from tests.utils import DataFrameMatcher
+
+
+def test_primary_key_detection_metrics():
```

Comment:
Since this is testing a class method, let's make this test inside a class as well.

```
class TestDetectionMetric:

    def test_compute(self):
        ...
```